apps/meteor/client/views/admin/permissions/EditRolePage.tsx

```diff
@@ -16,6 +17,8 @@ const EditRolePage = ({ role, isEnterprise }: { role?: IRole; isEnterprise: bool
 	const setModal = useSetModal();
 	const usersInRoleRouter = useRoute('admin-permissions');
 	const router = useRoute('admin-permissions');
+	const { data } = useIsEnterprise();
+	const isEnterprise = !!data?.isEnterprise;
```

Comment:
Is enterprise is already defined. @gabriellsh

Reply:
oops! 🙈 Thanks @rodrigok

Reply:
Actually I just discovered this came from develop, and didn't conflict since that line wasn't modified.